xde/src/xde.rs

```diff
@@ -2174,14 +2223,12 @@ struct MulticastRxContext<'a> {
 /// telling the switch which port groups to replicate to: External (front panel),
 /// Underlay (other sleds), or Both.
 ///
-/// `cpu_devs` may be None if the fast-path check indicated no local subscribers exist.
-///
 /// [`Replication`]: oxide_vpc::api::Replication
 fn handle_mcast_tx<'a>(
     ctx: MulticastTxContext,
     src_dev: &'a XdeDev,
     postbox: &mut TxPostbox,
-    cpu_devs: Option<&'a DevMap>,
+    cpu_devs: &'a DevMap,
```

Comment:
Nit: maybe `devs`, to make it clearer that this is not originated from the per-CPU state.